notebooks/.gitignore

```diff
@@ -36,3 +36,5 @@ MAST/Kepler/Kepler_TPF/mastDownload/
 !**/skyfile.txt
 !**/exclusions.txt
 !**/inclusions*.txt
+!STIS/contrast_sensitivity/ETC_example.jpg
```

Comment:
These files probably don't need to modify the general `.gitignore` file for everyone, especially since these files seem to be included in the repo.